packages/codemods/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@fluentui/codemods",
+  "version": "0.1.0",
+  "description": "package to enable easy upgrades to new versions",
+  "private": true,
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "bin": {
+    "upgrade": "./bin/upgrade.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
```

Comment:
You can remove this one if the package doesn't use snapshots (haven't finished looking yet)